src/tiling/tile_state.rs

```diff
@@ -126,24 +126,24 @@ impl IndexMut<usize> for MiTileState {
 impl<'a, T: Pixel> TileStateMut<'a, T> {
   pub fn new(
     fs: &'a mut FrameState<T>, sbo: PlaneSuperBlockOffset,
-    sb_size_log2: usize, width: usize, height: usize,
+    sb_size_log2: usize, w: usize, h: usize,
```

Comment:
It appears that reverting the rename of the `width` and `height` parameters will result in a smaller, more focused set of changes to this function.

Reply:
Yes, that is surely right. I also wanted to change as small as possible but it seems the lines 142 and 143 prohibits it by a rust grammer. I did end up compile errors. I thought one.possible way to keep formal parameter names width and height here would be change the field names width,height of TileRect struct.

Reply:
We could just keep as
Width.... instead of explicitly assigning right? 
I think if we are passing all args we could do like that 
If so it might work, not so sure

Reply:
By introducing two new variables to keep width and height that are rounded up to SB, I think I can keep existing width and height. Let me try..